ecc/bls12-377/bls12-377.go

```diff
@@ -148,3 +148,8 @@ func Generators() (g1Jac G1Jac, g2Jac G2Jac, g1Aff G1Affine, g2Aff G2Affine) {
 	g2Jac = g2Gen
 	return
 }
+
+// CurveCoefficients returns the a, b coefficients of the curve equation.
+func CurveCoefficients() (a, b fp.Element) {
```

Comment:
maybe define `aCurveCoeff` for all curve

Reply:
Good idea. Will do

Reply:
Added aCurveCoeff and return it in `CurveCoefficients` methods